apps/files_trashbin/lib/Command/ExpireTrash.php

```diff
@@ -74,7 +74,7 @@ protected function configure() {
 	protected function execute(InputInterface $input, OutputInterface $output): int {
 		$maxAge = $this->expiration->getMaxAgeAsTimestamp();
 		if (!$maxAge) {
-			$output->writeln("No expiry configured.");
+			$output->writeln("Auto expiration is configured - expiration will be handled automatically.");
```

Comment:
```suggestion
			$output->writeln("Auto expiration is configured - keeps files and folders in the trash bin for 30 days and automatically deletes anytime after that if space is needed (note: files may not be deleted if space is not needed)");
```
As described here https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/config_sample_php_parameters.html#deleted-items-trash-bin :)

Reply:
Much clearer :+1:

Reply:
Agree

Reply:
done